src/main/AndroidManifest.xml

```diff
@@ -41,21 +40,28 @@
     </activity>
     <activity android:name="EmbeddedBrowserActivity"
         android:screenOrientation="portrait"
-        android:configChanges="orientation|screenSize"/>
+        android:configChanges="orientation|screenSize"
+        tools:ignore="DiscouragedApi"/>
```

Comment:
The linting does not like that we are setting `screenOrientation="portrait"`....